contracts/issuance/IssuanceEth.sol

```diff
@@ -28,6 +29,7 @@ import "../state/StateMachine.sol";
 contract IssuanceEth is Ownable, StateMachine, ReentrancyGuard {
     using SafeMath for uint256;
     using FixidityLib for int256;
+    using SafeCast for *;
```

Comment:
Hmm, would it be possible to do like `using SafeCast for int256, uint256;`? Because `*` seems like, I can use it with `int8`.

Reply:
Yes, I thought about that one, however, `using SafeCast for int256, uint256;` does not compile. I guess it's ok since SafeCast might be extended for casting other types, too?

Reply:
hmm, I'm not sure of it is really necessary to others.
I any case, have you tried 
```
using SafeCast for int256;
using SafeCast for uint256;
```
?
It's just to keep it clear. If it does not work we leave it this way.